packages/infra/lib/ihe-gateway-v2-stack.ts

```diff
@@ -24,6 +24,7 @@ interface IHEGatewayV2LambdasNestedStackProps extends NestedStackProps {
   envType: EnvType;
   sentryDsn: string | undefined;
   iheResponsesBucketName: string;
+  iheXcpdResponsesBucketName: string;
```

Comment:
I suggest we name this `iheParsedResponsesBucketName` and use the same structure as we have for `iheResponsesBucketName`.. this way we can just change the bucket name and we can flip between raw/parsed responses.

I also considered whether we could use the same bucket and have Athena filter our `.xml` files when indexing it. Not sure if that would be helpful or not.

For reference, we already store diff steps/processing stages of a "file" on the same bucket, see the FHIR conversion one.

Reply:
The current structure of the responses bucket is not partitioned, which means using the data set with Athena won't be optimized. How critical is the similarity?

Reply:
What do you mean by partitioned? We prefix objects on S3. Isn't that enough for partitioning w/ Athena?

Reply:
Already discussed in the downstream PR.

Reply:
Done.